cmd/ejson2env/secrets.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+
+	"github.com/Shopify/ejson"
+)
+
+// ReadSecrets reads the secrets for the passed filename and
+// returns them as a map[string]interface{}.
+func ReadSecrets(filename, keyDir, privateKey string) (map[string]interface{}, error) {
+	var secrets map[string]interface{}
```

Comment:
Likely safer to `secrets := make(map[string]interface{})` here. Uninitialized maps are easy to cause panics with